services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -159,8 +174,8 @@ public final void init() {
       LOGGER.error(
           "Null resource assignment from HelixCustomizedViewOfflinePushRepository in ReadQuotaEnforcementHandler");
     } else {
-      for (String resource: customizedViewRepository.getResourceAssignment().getAssignedResources()) {
-        this.onExternalViewChange(customizedViewRepository.getPartitionAssignments(resource));
+      for (String resource: resourceAssignment.getAssignedResources()) {
+        this.onExternalViewChange(resourceAssignment.getPartitionAssignment(resource));
```

Comment:
I think that's because the code originally was written before CV existed. Let me see if it's feasible to completely rely on CV as trigger and not both.